hodometer/Dockerfile.receiver

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.20.11-alpine AS builder
+FROM golang:1.20.13-alpine AS builder
```

Comment:
Apologies for the long list of comments on rather trivial changes, but I'm lacking context here. I see `golang:*-bullseye`, `golang:*-alpine` and `golang:*` all used in those dockerfiles. We should probably aim to move to a single base unless there are good reasons to keep things different (just to elliminate possible divergences in the compiled binaries)

Generally this should not present a problem with go as we get staticly linked binaries, but if we're using CGO anywhere there will be a dependency on glibc which might differ between debian (bullseye) and redhat/alpine

Reply:
Some images require glibc, which alpine does not provide (based on some comments in the code). I think also the aim was to keep the image as small as possible. We probably can move them all to the same base image if the size difference is not significant. I will leave it to a future PR.